apps/builder/app/shared/copy-paste/plugin-webflow/schema.ts

```diff
@@ -76,8 +105,13 @@ export const WfElementNode = z.union([
     }),
   }),
 ]);
+
 export type WfElementNode = z.infer<typeof WfElementNode>;
 
+wfNodeTypes.forEach((nodeType) => {
+  nodeType satisfies WfElementNode["type"];
+});
```

Comment:
Is there a really need for runtime here?
```ts
const wfNodeTypes = [...] as const satisfies WfElementNode["type"][]
```

Reply:
I will try, didn't figure out how to write that